src/components/TabBar/TabBar.jsx

```diff
@@ -16,7 +16,7 @@ const TabBar = ({ router, tabs, activeTab }) => (
     value={activeTab}
   >
     {tabs.map((tab, index) => (
-      <Tab key={index} className={styles.tab} value={tab.route} label={tab.label} onActive={tab => onActive(tab, router)}>
+      <Tab key={index} className={styles.tab} value={tab.route} label={tab.label.replace(/\//g, '')} onActive={tab => onActive(tab, router)}>
```

Comment:
This just strips out the slashes for display purposes. I don't know if I actually need the slashes anymore. I'll take them out if I don't.